### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by leduytungframgia**

### CODE REVIEW

File: app/controllers/subjects_controller.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
 class SubjectsController < ApplicationController
   before_action :logged_in_user
-  
+
   def show
     @subject = Subject.find params[:id]
   end
+
+  def update
+    @subject = Subject.find params[:id]
```
Comment (nguyenduyta) [inline app/controllers/subjects_controller.rb]: viet thanh 1 action

File: app/views/supervisor/subjects/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<% provide :title, @subject.name %>
+<div class="panel panel-info">
+  <div class="alert alert-info" role="alert"><span class="user">
```
Comment (dieunb) [inline app/views/supervisor/subjects/show.html.erb]: missing `</div>`

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -8,15 +8,12 @@
...
-          <h4><%= us.subject.name %></h4>
-          <span class="label label-primary"><%= us.status %></span>
-          <%= us.created_at %>
-          <%= render us.subject.tasks %>
-        </li>
-      </div>
-    <% end %>
   </aside>
 </div>
+<% @user.activities.each do |ac| %>
```
Comment (dieunb) [inline app/views/users/show.html.erb]: Should use `render @user.activities`

File: app/views/supervisor/subjects/show.html.erb (hunk 2 above)
Comment (leduytungframgia) [inline app/views/supervisor/subjects/show.html.erb]: fixed

**Pull request closed without merging**